src/Plugin/SapiHandlerAnnotationInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\sapi\Plugin\SapiHandlerAnnotationInterface.
+ */
+
+namespace Drupal\sapi\Plugin;
+
+use Drupal\Component\Plugin\PluginInspectionInterface;
+
+/**
+ * Defines an interface for Sapi handler annotation plugins.
+ */
+interface SapiHandlerAnnotationInterface extends PluginInspectionInterface {
```

Comment:
Similarly to the comments above, I'd suggest to go with `StatisticsPluginInterface`.